lib/mutant/isolation.rb

```diff
@@ -33,30 +37,41 @@ module Fork
       #   if block terminates abnormal
       #
       # rubocop:disable MethodLength
-      def self.call(&block)
-        IO.pipe(binmode: true) do |reader, writer|
-          writer.binmode
+      def call(&block)
+        io.pipe(binmode: true) do |reader, writer|
           begin
```

Comment:
Passing a "data clump" is always a sign of a missing abstraction. Where in this case it could be a sign of "ruby not providing tuples".

For now I'll take the explicit approach.